### ISSUE TITLE
Not verified users are able to login and retrieve a token

### ISSUE THREAD

**Issue by bwqr** [bounswe/bounswe2019group6#146]
Hi there, I have found out a bug about login.

### Expected Behavior

Not verified users should not be able to login system. A **not verified error** should be thrown.

### Current Behavior

While sending a valid username and password to login, we can receive a token even user did not verify his email.

### Possible Solution

Check if user is verified after authentication of user.

### Steps to Reproduce

1. Signup to the system.
2. Do not verify your account by going to url which came to your email.
3. Try to login into system.
4. Receive a valid token.

### Possible Implementation
<!--- Not obligatory, bu

**Comment by enozcan** [comment]
Hi @bwqr 

The bug is fixed on 2332980c0574acfe7e1cacde00c699438db7e1c7 and the fixed version will be deployed asap. Thanks for the report.

**Comment by enozcan** [comment]
The new response is as follows:

```json
{
    "timestamp": 1571946042451,
    "status": 412,
    "error": "Precondition Failed",
    "message": "Account is not verified.",
    "path": "/login"
}
```
